cumulusci/robotframework/Salesforce.py

```diff
@@ -231,32 +231,32 @@ def _get_related_list_count(self, locator):
     def go_to_object_home(self, obj_name):
         """ Navigates to the Home view of a Salesforce Object """
         url = self.cumulusci.org.lightning_base_url
-        url = "{}/one/one.app#/sObject/{}/home".format(url, obj_name)
+        url = "{}/lightning/o/{}/home".format(url, obj_name)
         self.selenium.go_to(url)
 
     def go_to_object_list(self, obj_name, filter_name=None):
         """ Navigates to the Home view of a Salesforce Object """
         url = self.cumulusci.org.lightning_base_url
-        url = "{}/one/one.app#/sObject/{}/list".format(url, obj_name)
+        url = "{}/lightning/o/{}/list".format(url, obj_name)
         if filter_name:
             url += "?filterName={}".format(filter_name)
         self.selenium.go_to(url)
 
-    def go_to_record_home(self, obj_id, filter_name=None):
+    def go_to_record_home(self, obj_name, obj_id):
         """ Navigates to the Home view of a Salesforce Object """
         url = self.cumulusci.org.lightning_base_url
-        url = "{}/one/one.app#/sObject/{}/view".format(url, obj_id)
+        url = "{}/lightning/r/{}/{}/view".format(url, obj_name, obj_id)
```

Comment:
Can we use `"{}/{}".format(url, obj_id)` to avoid breaking existing tests using go_to_record_home?